libraries/networking/src/ResourceCache.cpp

```diff
@@ -809,6 +807,14 @@ void Resource::handleReplyFinished() {
         }
 
         auto data = _request->getData();
+        if (_request->getUrl().scheme() == "qrc") {
+            // For resources under qrc://, there's no actual download being done, so
+            // handleDownloadProgress never gets called. We get the full length here
+            // at the end.
+            _bytesTotal = data.length();
+        }
+
+        setSize(_bytesTotal);
```

Comment:
Note we're not calling setSize in the case of failure anymore. 

I think this should be fine since on failure no data arrives, and this can cause confusion in ResourceCache.